scripts.js

```diff
@@ -41,14 +49,12 @@ function getCanvasCtxFromSelector(selector) {
   return inputCanvas.getContext('2d');
 }
 
-function messWithPixels(pixels) {
-  for (let i = 0; i < pixels.data.length; i+=4) {
-    pixels.data[i + 0] = pixels.data[i + 0] - 50; // RED
-    pixels.data[i + 1] = pixels.data[i + 1] - 50; // GREEN
-    pixels.data[i + 2] = pixels.data[i + 2] + 100; // Blue
-  }
-  
-  return pixels;
+function brightenPixels({width, height, data}) {
+  return new ImageData(
+    data.map( (pixel,index) => index % 4 !== 3 ? pixel + 20 : pixel ),
```

Comment:
Pixel data which appears to be a Uint8ClampedArray, but what i can't seem to get an answer is if it is possible to send it such a large image that the index would cross over the largest int allowed in js seeing it is all. I'm assuming now but would be interesting to know if such a feat is achievable.

Reply:
@khutten sorry above comment was for you.

Reply:
I'm not sure @mrhut10, I don't know anything about larges allowable in js, is that based on size (memory) or size (length), because if it's the former having an 8-bit array would definitely help.

On a seperate note, the reason I'm using a `new ImageData` is because I'm trying not at mutate the incoming object, not exactly sure what I'm doing but trying my best 🤷‍♀️ 
Also do you like the ES6 destructuring here?